test/result.spec.ts

```diff
@@ -57,10 +59,10 @@ describe("#result() UT", () => {
 
     try {
       await scorecard.result(expectedRepository);
-      expect(true).to.equal(false, "the test should never execute this assertion");
+      assert.strictEqual(true,false, "the test should never execute this assertion");
     }
     catch (e) {
-      expect(e.message, "Not Found");
+      assert(e.message, "Not Found")
     }
```

Comment:
Maybe we can use assert.rejects here? (I think we don't do this in tape cause that's not available).